testing/helpers/native-helpers.bash

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Couchbase, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file  except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the  License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Starts a Couchbase cluster in Vagrant, featuring couchbase-exporter and couchbase-fluent-bit configured.
+# Uses the variables VAGRANT_HOST, VAGRANT_CPUS, and VAGRANT_RAM to determine
+# the cluster configuration, and CMOS_HOST as the Fluent Bit Loki destination.
+# Parameters:
+# $1: The Couchbase Server version to test (default: 6.6.3)
+# $2: The OS to use (default: centos7)
+function start_vagrant_cluster() {
+    local cb_version=${1:-6.6.3}
+    local os=${2:-centos7}
+    # Check if requirements are installed
+    if ! command -v vagrant &> /dev/null
+    then
+        echo "Vagrant not installed!"
+        exit 1
+    fi
+    if ! command -v ansible &> /dev/null
```

Comment:
Ah right, yeah need to capture this in the dependencies

Reply:
I'll file a follow-up PR against the README once this is in, because this branch has outdated docs infra